sorc/ncep_post.fd/CALWXT.f

```diff
@@ -247,6 +262,8 @@ SUBROUTINE CALWXT_POST(T,Q,PMID,PINT,HTM,LMH,PREC,ZINT,IWX,ZWET)
           DZKL=T(I,J,L)*(Q(I,J,L)*D608+H1)*ROG*ALOG(PINTK1/PM150)
         AREA1=(TWET(I,J,L)-273.15)*DZKL
         AREAS8=AREAS8+AREA1
+        exit looppintk1
```

Comment:
@BoCui-NOAA Why another "exit" before the do loop end?

Reply:
Do you mean the statement "exit looppintk". It is like a label to show where the exit will stop.